src/block_cache/block_chunkcache.c

```diff
@@ -261,6 +261,37 @@ __chunkcache_eviction_thread(void *arg)
  * __wt_chunkcache_get --
  *     If the cache has the data at the given size and offset, copy it into the supplied buffer.
  *     Otherwise, read and cache the chunks containing the requested data.
+ *
```

Comment:
I'm flexible on this, but I think it would be better to attach this to the struct definition rather than this function. Most of the content here isn't specific to `__wt_chunkcache_get`, and I suspect people are more likely to end up looking at the lock declarations in the chunk cache structs when looking for guidance on how to use them (but perhaps that's just me).

Reply:
I agree with @korteland's comment - it's more of a comment on how the data structures are used, so probably better on the struct.